x-pack/plugins/ml/public/application/explorer/explorer_charts/_explorer_chart.scss

```diff
@@ -12,13 +12,6 @@
         opacity: 1;
       }
 
-      rect.selected-interval {
```

Comment:
This PR might be a good time to make the styling of the lines and marker consistent with the changes that were recently made for the Single Metric Viewer in #93581. Or add it to the meta issue #94650.